lib/gollum/templates/layout.mustache

```diff
@@ -8,13 +8,13 @@
   {{#sprockets_stylesheet_tag}}app{{/sprockets_stylesheet_tag}}
   {{#sprockets_stylesheet_tag}}print print{{/sprockets_stylesheet_tag}}
 
-  {{#css}}<link rel="stylesheet" type="text/css" href="{{custom_path}}/custom.css" media="all">{{/css}}
+  {{#css}}<link rel="stylesheet" type="text/css" href="{{custom_css}}" media="all">{{/css}}
   {{#noindex}}<meta name="robots" content="noindex, nofollow" />{{/noindex}}
 
 
   <script>
   var criticMarkup = '{{critic_markup}}';
-	var baseUrl = '{{base_url}}';
+	var baseUrl = '{{page_route}}';
```

Comment:
I find `page_route` here a bit confusing, as `base_url` is conceptually independent from pages. Could we keep the `base_url` helper, perhaps as an alias of `custom_path`, and use it here?

Reply:
> I find `page_route` here a bit confusing

Agreed. But I may have a better idea. We can replace the javascript code in the following lines with a template variable and remove `baseUrl`, i.e., `uploadDest + {{url_tail}}`.
https://github.com/gollum/gollum/blob/6e6d9d8c1fbeca072c2b7a14208d808757bd5b44/lib/gollum/templates/layout.mustache#L20-L22

Reply:
No, `baseUrl` seems to be widely used in `gollum.js.erb`. Let's just restore `base_url` for now.

Reply:
Committed. I kept `custom_path` separate from `base_url` because some user might happen to be using it to customize the path to `custom.css` and `custom.js`.